markets/aave/commons.ts

```diff
@@ -209,6 +210,7 @@ export const CommonsConfig: ICommonConfiguration = {
       SNX: '0xF9A76ae7a1075Fe7d646b06fF05Bd48b9FA5582e',
       SUSD: '0xb343e7a1aF578FA35632435243D814e7497622f7',
       TUSD: '0x7aeCF1c19661d12E962b69eBC8f6b2E63a55C660',
+      UMA: '0x489Bf230d4Ab5c2083556E394a28276C22c3B580',
```

Comment:
This should be the CL price feed (ETH denominated): 0xf817B69EA583CAFF291E287CaE00Ea329d22765C

Reply:
Deleted this line